client/src/pages/HomePage.js

```diff
@@ -110,24 +110,26 @@ class HomePage extends Component {
           ) : (
             <>
               {toPlayChallenges.map((value, index) => (
-                <article key={index}>
-                  <Avatar
-                    size="75px"
-                    eyes={usersData[value.opponent]?.avatar.eyes}
-                    hands={usersData[value.opponent]?.avatar.hands}
-                    hat={usersData[value.opponent]?.avatar.hat}
-                    mouth={usersData[value.opponent]?.avatar.mouth}
-                    colorBG={usersData[value.opponent]?.avatar.colorBG}
-                    colorBody={usersData[value.opponent]?.avatar.colorBody}
-                  />
-                  <Link to={`/duel/${value.id}`} className="challenges-text">
-                    <div>
-                      <h3>{value.opponent}</h3>
-                      <p>Vous pouvez jouer le round {value.currentRound}</p>
+                <Link to={`/duel/${value.id}`} key={index}>
+                  <article>
+                    <Avatar
+                      size="75px"
+                      eyes={usersData[value.opponent]?.avatar.eyes}
+                      hands={usersData[value.opponent]?.avatar.hands}
+                      hat={usersData[value.opponent]?.avatar.hat}
+                      mouth={usersData[value.opponent]?.avatar.mouth}
+                      colorBG={usersData[value.opponent]?.avatar.colorBG}
+                      colorBody={usersData[value.opponent]?.avatar.colorBody}
+                    />
+                    <div className="challenges-text">
+                      <div>
+                        <h3>{value.opponent}</h3>
+                        <p>Vous pouvez jouer le round {value.currentRound}</p>
+                      </div>
+                      <ChevronRightIcon />
                     </div>
-                    <ChevronRightIcon />
-                  </Link>
-                </article>
+                  </article>
+                </Link>
```

Comment:
It was intented to not put the avatar in the link: later, we may add a link on the avatar to go to the profile page of the user

Reply:
Oh okay!

Reply:
I think we'll have (to access to user statistics, view the full name of someone, and maybe add informations like the class group (to differenciate in case of 2 people with the same pseudo with a number).
I think the rectangle is big enough to click on

Reply:
Okay!